readthedocs.yaml

```
@@ -1,13 +1,15 @@
 # Read the Docs configuration file
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
```

Try this & see if it still works or not;

```suggestion
    python: "3.12"
```